src/njoy21/input/DTFR/Card3a/test/Card3a.test.cpp

```diff
@@ -0,0 +1,43 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "Validating card3a inputs",
+  "[DFTR], [Card3a]" ){
+  GIVEN( "valid DTFR card3a inputs" ){
+    WHEN( "all DTFR inputs provided" ){
+      iRecordStream<char> iss( std::istringstream( "221 0 0" ) );
```

Comment:
If you specify the default value, you don't know if what is returned is what is read in, or is the default value. If you specify something else, you can be certain.